### DISCUSSION TITLE
Does v5 support Nuxt Bridge?

### DISCUSSION THREAD

**Discussion by bmulholland** [nuxt-community/apollo-module#454] [Q&A]
I'm trying to figure out my upgrade path to Nuxt 3. Can I upgrade this without upgrading from Nuxt Bridge to Nuxt 3? Ideally I wouldn't have to change so much at once, so decoupling those two upgrades would be very helpful.

**Comment by DenFin** [comment]
@bmulholland we are currently in the same situation, figuring out a path to upgrade to Nuxt 3. did you ever try out v5 with Nuxt Bridge? did you experience any major problems?

**Reply by bmulholland** [reply]
We upgraded to bridge and with something similar to Apollo v5, but it’s been a bit customized so I wouldn’t say for sure.
